app/authentication.html

```diff
@@ -66,5 +66,6 @@
   <script src="js/sumo_db.js" defer></script>
   <script src="js/authentication_controller.js" defer></script>
   <script src="js/error_controller.js" defer></script>
+  <script src="js/spinner_controller.js" defer></script>
```

Comment:
We need error_controller and spinner_controller to be included before the main controller for each view. In question.html, because question_controller.js is included before spinner_controller, we are missing the request-start event.